packages/client/hmi-client/src/utils/data-util.ts

```diff
@@ -131,3 +139,11 @@ export function extensionFromProgrammingLanguage(lang: ProgrammingLanguage): str
 			return null;
 	}
 }
+
+export function isInitial(obj: Initial | ModelParameter | null): obj is Initial {
+	return obj !== null && 'target' in obj && 'expression' in obj && 'expression_mathml' in obj;
+}
+
+export function isModelParameter(obj: Initial | ModelParameter | null): obj is ModelParameter {
+	return obj !== null && 'id' in obj;
+}
```

Comment:
We have several a couple of different places for model-related utilities that are more appropriate.

- services/model, or
- model-representation/_stuff_under_there

Reply:
I would like to revisit the whole model page that @blanchco spent the last few weeks hacking together for the Evaluation needs. There is so much refactor and cleaning that needs to happens.
I think before doing this, we need a proper _What is a model in Terarium_ discussion as a team

Reply:
Yes, but is a call to contain the ugliness and not an exemption or an excuse to create additional distractions.

Reply:
I've moved these functions to `services/model`